src/rendering/luaconsole.cpp

```diff
@@ -507,20 +508,11 @@ bool LuaConsole::keyboardCallback(Key key, KeyModifier modifier, KeyAction actio
             const size_t fullLength = _autoCompleteInfo.initialValue.length();
             const bool correctLength = command.length() >= fullLength;
 
-            std::string commandLowerCase;
```

Comment:
@alexanderbock does this change introduce an undesired performance penalty, do you think?

It seems to be the only place where we did not first copy the string explicitly

Reply:
My gut feeling is that it is probably faster since we are modifying the string in place rather then back_inserting it.

Either way it's the Lua console, so performance is not really _that_ important here anyway